server.js

```diff
@@ -49,10 +53,16 @@ app.post('/api/v1/favorites', (request, response) => {
 });
 
 app.delete('/api/v1/favorites/:isbn', (request, response) => {
-  queries.removeBookFromFavs(request);
-  response.status(200).json({
-    message: `Book with isbn #${request.params.isbn} has been deleted from favorites`
-  })
+  queries.removeBookFromFavs(request)
+    .then(count => {
+      if (count) {
```

Comment:
Great logic! Need to target the count (count of deleted row) vs the favorites array itself.